snr.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import astropy.units as u
+
+
+def snr(counts, npix=1, nb=np.inf, background=0, darkcurrent=0,
+        readnoise=0, gain=0, ADerr=0.289):
+    """ A function to calculate the signal to noise ratio (SNR) of an
+    astronomical observation.
+
+    Parameters
+    ----------
+    counts (float or int)
```

Comment:
This is close to, but not quite, the format of docstrings in the astropy community. See for example [Time](http://docs.astropy.org/en/stable/_modules/astropy/time/core.html#Time).